integration/simulation/simulation_test.go

```diff
@@ -49,7 +49,7 @@ func TestSimulation(t *testing.T) {
 		l2NetworkCfg,
 		avgBlockDurationUSecs,
 		avgGossipPeriod,
-		false,
+		true,
```

Comment:
On purpose?

Reply:
bug - fixed now :)

Reply:
let's put this back to false, and change it locally.
We'll have to create another simulation for dev use

Reply:
Yeah that's set to false now :)